apps/editing-toolkit/editing-toolkit-plugin/dotcom-fse/plugins/editor-template-classes/index.js

```diff
@@ -23,21 +24,26 @@ const EditorTemplateClasses = withSelect( ( select ) => {
 	} );
 	return { templateClasses };
 } )( ( { templateClasses } ) => {
-	const blockListInception = setInterval( () => {
-		const blockListParent = document.querySelector( '.block-editor-writing-flow' );
+	useEffect( () => {
+		const blockListInception = setInterval( () => {
```

Comment:
Im just realizing we haven't (nor did we previously) set any delta time for this interval.  Do you think we should add a short timer so it isn't running every 10ms until the dom is ready with what this is looking for?

Reply:
Great point.  I added a 100ms interval.